processors/metadata.go

```diff
@@ -0,0 +1,30 @@
+package processors
+
+import "github.com/algorand/indexer/plugins"
+
+// ProcessorMetadata returns fields relevant to identification and description of plugins.
+type ProcessorMetadata struct {
+	ImpName        string
+	ImpDescription string
+	ImpDeprecated  bool
```

Comment:
Not sure if "Imp" here stands for "Implementation", but I think it's a bit confusing considering we're going to have "Importers" as well.

Reply:
+1, I think it's fine to go with Name/Description/Deprecated

Reply:
Name wont work since it is also a function for ProcessorMetadata.  Will go with "ImplementationName" etc